modules/service-accounts-using-credentials-inside-a-container.adoc

```diff
@@ -26,15 +26,13 @@ by the token:
 +
 ----
 $ curl --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
-    "https://openshift.default.svc.cluster.local/oapi/v1/users/~" \
+    "https://openshift.default.svc.cluster.local/apis/core/v1/users/~" \
```

Comment:
AFAIK， core is a api group name, not the path of the api. 
`oc get --raw "/apis" | grep '/core'`
no results
```
$ oc get --raw "/" | grep user
    "/apis/user.openshift.io",
    "/apis/user.openshift.io/v1",

For example,
$ token=$(oc whoami -t)
$ curl -X GET https://<openshift-kube-apiserver>:6443/apis/user.openshift.io/v1/users/~ --header "Authorization: Bearer $token" --insecure
{
  "kind": "User",
  "apiVersion": "user.openshift.io/v1",
  "metadata": {
    "name": "kube:admin",
    "selfLink": "/apis/user.openshift.io/v1/users/kube%3Aadmin",
    "creationTimestamp": null
  },
  "identities": null,
  "groups": [
    "system:authenticated",
    "system:cluster-admins"
  ]
}
```